Include/pyport.h

```diff
@@ -8,6 +8,22 @@
 #  error "Python's source code assumes C's unsigned char is an 8-bit type"
 #endif
 
+/*
+ * Disable keywords and operators that are not recognized by CLion.
+ *
+ * This is only a temporary solution until CLion correctly supports them.
+ */
+#ifdef __JETBRAINS_IDE__
+/*
+ * Prevents false positives in CLion's static analysis which incorrectly
+ * detects _Atomic(size_t) and _Atomic(uintptr_t) as duplicated declarations
+ * of size_t and uintptr_t respectively.
+ */
+#define _Atomic(x)  x
```

Comment:
Why not simply `#define _Atomic`?

Reply:
(Sorry, but this is really the only workaround I was able to find...)

Reply:
Oh actually, it's because it's not legal C as well. Replacing `_Atomic` only would give `extern (int) something;` for instance, but this is not valid in C.